### ISSUE TITLE
Set selected country

### ISSUE THREAD

**Issue by jrock2004** [tableau-mkt/jquery.addressfield#58]
What is the best way to set which country should be selected on page load? Do I just write my own JS to do this? If I just need to set the selected element after load, is there a method that I can have it redraw the fields?

**Comment by iamEAP** [comment]
Hmm...  This will depend on how you're using the plugin:

0. If you're applying the plugin to a form that already has a populated country select list, then there's nothing extra you need to do. Just set the `selected` attribute on the option element in HTML and it'll work as expected (if this isn't how you're loading the plugin now, consider doing it this way as one of the easier ways to accomplish this).
0. If you're using the plugin itself to generate the country select list, you're right, as it is now, you'll have to manually set the selected country in JavaScript (though a PR that let you set it via a property in the initialization config object would be sweet).

Something like the following:

```javascript
// Bind to the addressfield:after event, but only execute once.
$('#your-address-form-container').one('addressfield:after', function setDefaultCountry() {
  // Set France as default country. Change method call guarantees address field re-render.
  $('#country').val('FR').change();
});
```

**Comment by iamEAP** [comment]
Fixed in #60.
